iron-icons.html

```diff
@@ -23,14 +23,10 @@
 
     <iron-icon icon="maps:directions-bus"></iron-icon>
 
-To load a subset of icons from one of the default `iron-icons` sets, you can
```

Comment:
Ugh, I realize the script removes this but can you maybe manually undo these changes?